### ISSUE TITLE
Add gentrans

### ISSUE THREAD

**Issue by njam** [cargomedia/debian-packages#15]
http://gentrans.sourceforge.net/

```
git://git.code.sf.net/p/gentrans/code
cd gst-entrans
git fetch && git checkout RELEASE-1.0.3
make clean
./autogen.sh >../autogen-gst-entrans.out
./configure --prefix=/usr/local
make
make install
cd -
```

**Comment by dergraf** [comment]
working on it

**Comment by dergraf** [comment]
if plugins install themselves into /usr/lib/gstreamer-1.0 require that the ``GST_PLUGIN_SYSTEM_PATH_1_0=/usr/lib/gstreamer-1.0`` is properly set.
http://gstreamer.freedesktop.org/data/doc/gstreamer/head/gstreamer/html/gst-running.html


Trying to figure out if the plugins can be installed in to /usr/lib directly.

**Comment by ppp0** [comment]
solved by #37
